packages/astro/src/content/runtime.ts

```diff
@@ -404,6 +458,12 @@ export function createGetEntry({
 			if (!entry) {
 				return;
 			}
+
+			const { schema } = liveCollections[collection];
+			if (schema) {
```

Comment:
Shouldn't we check if live collections are enabled?

Reply:
It's inside a check for whether the collection is in `liveCollections`, which is only populated if live collections are enabled